lang/lang.txt

```diff
@@ -17144,7 +17144,7 @@ fr=Nombre total de pannes de disque accumulées sur toute sa durée de vie.\nCha
 de=Gesamtzahl der über die gesamte Lebensdauer kumulierten Festplattenausfälle.\nJede Festplatte definiert eine Reihe von Attributen, ihre tatsächlichen Werte und Schwellwerte. Wenn ein Attribut unter den definierten Schwellwert fällt, wird es als Fehler betrachtet.\nBei NVMe-Festplatten werden die Gerätezuverlässigkeit verschlechtert oder Medien im schreibgeschützten Modus als Fehlerbedingungen betrachtet.\nWenn ein Fehler erkannt wird, liefert ein zusätzlicher Sensorwert mehr genaue Angaben. Auf dem HWiNFO-Hauptbildschirm finden Sie alle Details zu jedem Attribut.
 es=Número total de fallos de la disco acumulados durante toda su vida útil.\nCada disco define un conjunto de atributos, sus valores reales y umbrales. Cuando un atributo cae por debajo del umbral definido, se considera una falla.\nPara las disco NVMe, los estados de Confiabilidad del dispositivo degradada o Medios en modo de solo lectura se consideran condiciones de falla.\nCuando se detecta una falla, un valor de sensor adicional proporcionará más detalles precisos. Consulte la pantalla principal de HWiNFO para obtener detalles completos sobre cada atributo.
 pt-BR=Número total de falhas de disco acumuladas durante toda a sua vida útil.\nCada disco define um conjunto de atributos, seus valores reais e limites. Quando um atributo cai abaixo do limite definido, ele é considerado uma falha.\nPara o disco NVMe, os estados de Confiabilidade do dispositivo degradada ou Mídia no modo somente leitura são considerados como condições de falha.\nQuando uma falha é detectada, um valor de sensor adicional fornecerá mais detalhes precisos. Verifique a tela principal do HWiNFO para detalhes completos sobre cada atributo.
-it=Numero totale di guasti dell'disco accumulati durante l'intero ciclo di vita.\nOgni disco definisce un insieme di attributi, i loro valori effettivi e le soglie. Quando un attributo scende al di sotto della soglia definita viene considerato un errore.\nPer le disco NVMe, gli stati di Affidabilità del dispositivo ridotta o Supporti in modalità di sola lettura sono considerati condizioni di errore.\nQuando viene rilevato un errore, un valore del sensore aggiuntivo fornirà più dettagli precisi. Controlla la schermata principale di HWiNFO per tutti i dettagli su ciascun attributo.
+it=Numero totale di guasti del                                                    disco accumulati durante l'intero ciclo di vita.\nOgni disco definisce un insieme di attributi, i loro valori effettivi e le soglie. Quando un attributo scende al di sotto della soglia definita viene considerato un errore.\nPer le disco NVMe, gli stati di Affidabilità del dispositivo ridotta o Supporti in modalità di sola lettura sono considerati condizioni di errore.\nQuando viene rilevato un errore, un valore del sensore aggiuntivo fornirà più dettagli precisi. Controlla la schermata principale di HWiNFO per tutti i dettagli su ciascun attributo.

```

Comment:
Is this intentional?

Reply:
No. Will be fixed in enxt language release. Thanks.